src/deluge/gui/menu_item/sequence/direction.h

```diff
@@ -37,6 +38,20 @@ class Direction final : public Selection {
 				return clip->getNoteRowForDrum(modelStack, kit->selectedDrum); // Still might be NULL;
 			}
 		}
+		else if (clip->output->type != OutputType::KIT) {
+			if (soundEditor.selectedNoteRow) {
+				// get model stack with note row but don't create note row if it doesn't exist
+				ModelStackWithNoteRow* modelStackWithNoteRow =
+				    clip->getNoteRowOnScreen(instrumentClipView.lastAuditionedYDisplay, modelStack);
+				if (modelStackWithNoteRow != nullptr) {
```

Comment:
this can never be null, it'll be the original modelstack with a null note row if it fails

You check that in the next line so this one can be removed entirely